src/NeoSharp.Core/Blockchain/Blockchain.cs

```diff
@@ -109,93 +119,20 @@ static int TransactionComparer(Stamp<Transaction> a, Stamp<Transaction> b)
         #region Blocks
 
         /// <inheritdoc />
-        public Task<bool> AddBlock(Block block)
+        public void AddBlock(Block block)
```

Comment:
another method that I see no reason to live in BlockChain class and IBlockChain interface. Who need this in the rest of the application should use the IBlockProcessor interface.